charts/prometheus-adapter/README.md

```diff
@@ -52,6 +52,10 @@ $ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 3.0.0
+
+Due to a change in deployment labels, the upgrade requires `helm upgrade --force` in order to re-create the deployment.
```

Comment:
Update would be still possible via: https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#from-chart-versions--500